eslint/eslint-patch/src/eslint-bulk-suppressions/bulk-suppressions-file.ts

```diff
@@ -25,11 +25,18 @@ const IS_RUNNING_IN_VSCODE: boolean = process.env[VSCODE_PID_ENV_VAR_NAME] !== u
 const TEN_SECONDS_MS: number = 10 * 1000;
 const SUPPRESSIONS_JSON_FILENAME: string = '.eslint-bulk-suppressions.json';
 
+function throwIfAnythingOtherThanNotExistError(e: NodeJS.ErrnoException): void | never {
+  if (e?.code !== 'ENOENT') {
```

Comment:
I believe you should be able to use the method from `FileSystem.isNotFoundError` to check this, though I think this script is intended to be lightweight so maybe bundling that would be a bit much.